charts/prefect-worker/templates/NOTES.txt

```diff
@@ -1 +1 @@
-1. Check Prefect worker connections in the prefect UI at {{ template "worker.appUrl" . }}
```

Comment:
removing the specific printout of the UI url -- we were taking in values for this for the solve purpose of constructing this value in the notes. I don't think thats necessary, so I'm opting to remove it